src/app/blog/[slug]/page.tsx

```diff
@@ -0,0 +1,23 @@
+import { compileMDX } from "next-mdx-remote/rsc";
+import { readFile } from "node:fs/promises";
+import { join } from "node:path";
+import { cwd } from "node:process";
+
+export type BlogParam = {
+  params: {
+    slug: string;
+  };
+};
+
+export default async function Page({ params: { slug } }: BlogParam) {
+  const filename = await readFile(
```

Comment:
move it to another variable for the sake of readibility

```ts
const filePath = join(cwd(), "content", `${slug}.md`)
const fileName = await readFile(filePath, "utf-8")
```

And readFile should return the content right ? instead of fileName ?
```ts 
const filePath = join(cwd(), "content", `${slug}.md`)
const fileContent = await readFile(filePath, "utf-8")
```